### PR TITLE
Changes from downstream

### PR DESCRIPTION
**Pull request by waybackarchiver**
Hello there.

Thank you to @RadhiFadlillah for taking the time to develop obelisk. Because of our scope of work, we are also maintaining obelisk and adding several of the new features.

The goal of this merge request is to tell everyone that we have an active downstream branch, so if you are interested, please feel free to contribute.

Thank you also to @hellodword for his contribution.

### CODE REVIEW

**Comment by fmartingr** [comment]
Hey @waybackarchiver, first of all, amazing work both of you :)

Now that I've stepped up as maintaner here, are you considering going back to the upstream project or are you planning to keep maintaining the fork for your uses anyway? I can't possibly maintain all go-shiori's repositories myself and keep a good pace, and you are already familiar enough with Obelisk.

Let me know what you think!

**Comment by waybackarchiver** [comment]
It’s cool to hear this.

If the upstream remains active, I will also merge changes back to it in the future.

File: README.md [hunk 1]
```diff
@@ -25,10 +25,15 @@ Obelisk is a Go package and CLI tool for saving web page as single HTML file, wi
...
 Run following command inside your Go project :
 
-```
+```shell
 go get -u -v github.com/go-shiori/obelisk
 ```
 
+If you prefer to use changes from this repository, run :
+```shell
+go mod edit -replace github.com/go-shiori/obelisk=github.com/wabarc/obelisk@main
```
Comment (fmartingr) [inline README.md:34]: This needs to be removed once merged

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+    - name: Write Go module list
+      run: go list -json -m all > go.list
+
+    - name: Perform Nancy
+      uses: sonatype-nexus-community/nancy-github-action@main
+      continue-on-error: true
+
+  fossa:
+    if: github.repository == 'wabarc/obelisk'
```
Comment (fmartingr) [inline .github/workflows/codeql-analysis.yml:120]: This need to be removed after merge

**Comment by fmartingr** [comment]
> It’s cool to hear this.
> If the upstream remains active, I will also merge changes back to it in the future.

Is your use case very specific? How do you feel about maintaining this repository?

**Comment by waybackarchiver** [comment]
Sorry for not responding sooner. I used obelisk in the [wayback](/wabarc/wayback) related projects, and there are some features, essentially minor changes, that I believe match the upstream features.

File: README.md (hunk 1 above)
Comment (waybackarchiver) [reply README.md]: done

File: .github/workflows/codeql-analysis.yml (hunk 2 above)
Comment (waybackarchiver) [reply .github/workflows/codeql-analysis.yml]: done

**Comment by fmartingr** [comment]
> Sorry for not responding sooner. I used obelisk in the [wayback](/wabarc/wayback) related projects, and there are some features, essentially minor changes, that I believe match the upstream features.

I was asking because I could gave you (and @hellodword) permissions on this repository if you feel like maintaining it, since you are dogfooding it and making improvements. But I don't know if you folks want to maintain upstream or if your changes requires deviation from the original in any way. Let me know what you think.

**Comment by waybackarchiver** [comment]
Our changes should not deviate from the upstream, and it would be great if we could be granted permission to maintain the upstream.

**Comment by waybackarchiver** [comment]
>> Sorry for not responding sooner. I used obelisk in the [wayback](https://github.com/wabarc/wayback) related projects, and there are some features, essentially minor changes, that I believe match the upstream features.
>
> I was asking because I could gave you (and @hellodword) permissions on this repository if you feel like maintaining it, since you are dogfooding it and making improvements. But I don't know if you folks want to maintain upstream or if your changes requires deviation from the original in any way. Let me know what you think.

And, what's your thoughts? @hellodword

**Comment by hellodword** [comment]
@waybackarchiver @fmartingr Glad to hear that this repository becomes the upstream again. :clinking_glasses: 

Thanks but for some privacy reasons, my github account could not be a member of any organization, sorry.

I will contribute to this repository by creating issue and pull-requests in the future. :smiley:

**Comment by fmartingr** [comment]
That's fair! 

I've invited @waybackarchiver to the repo. Please confirm that everything is alright.

I hope this speeds up development for this project :)

If you require anything related to the org/repo and want to reach me privately, you can use the mail in my github profile. I'll try and reach back ASAP.

**Comment by waybackarchiver** [comment]
@fmartingr Thank you for the invitation! I'm thrilled to be a part of this project, and I'll do everything I can to help it make progress.

**Comment by waybackarchiver** [comment]
Now that it's all been done, let us merge it.

**Pull request merged**